SpatialGDK/Documentation/content/get-started/build-unreal-fork.md

```diff
@@ -71,28 +71,45 @@ In File Explorer, navigate to the root directory of your clone of the SpatialOS
 In the same directory, double-click **GenerateProjectFiles.bat**.
 
 ### Step 5: Clone and install the plugin
-You need to clone the SpatialOS GDK plugin and install it in the UE fork and Example Project directory. You can follow either auto-install or manual-install, we recommend the auto-install. 
+You need to clone the SpatialOS GDK plugin and install it into the UE fork and Example Project directory. We recommend that you follow the auto-install method to reduce the number of manual steps needed  to get set up. The auto-install installs the GDK as an Engine plugin so you won't have to clone the GDK for each new project you set up.
 
-This method will reduce the number of manual steps you have to do to get set up, and will install up the GDK as an Engine plugin rather than a Project plugin. This means you won't have to clone the GDK for each new project you set up.
+There are two branches of the plugin repository and the Example Project: preview and release.  We recommend installing from the default release branch of both, which the auto-install below does for you, and the manual-install guides you to do.  
```

Comment:
Found the long sentence with three topics a bit of a stumble, so suggested breaking it up. Your call.

```suggestion
There are two branches of the plugin repository and the Example Project: preview and release.  We recommend installing from the default release branch of both. The auto-install does this for you, and the manual-install guides you to do this.  
```